src/helpers.ts

```diff
@@ -1,11 +1,16 @@
 import { BigNumber } from "@ethersproject/bignumber";
 
-import { Integer, SdkError, Usdc } from "./types";
+import { Address, Integer, SdkError, Usdc } from "./types";
 
 export const ZeroAddress = "0x0000000000000000000000000000000000000000";
 export const EthAddress = "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE";
 export const WethAddress = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
 
+// Returns truthy if address is defined as a native token address of a network
+export function isNativeToken(address: Address): boolean {
+  return address === EthAddress || address === ZeroAddress;
+}
```

Comment:
Can we add a test for this? I think `helpers.spec.ts` already exists, as we might want to add more to this and perhaps can already be refactored to something like:

```suggestion
export function isNativeToken(address: Address): boolean {
  return [EthAddress, ZeroAddress].includes(address);
}
```

Reply:
added test and refactored here https://github.com/yearn/yearn-sdk/pull/266/commits/7877a0603285abdfa2df919ca32c4bcf6d1d193f

Reply:
Nice! Thanks 🙇